packages/sitecore-jss-cli/src/scripts/create.ts

```diff
@@ -1,221 +1,29 @@
-import spawn from '../spawn';
 
-/**
- * @param {any} yargs
- */
-export function builder(yargs: Argv) {
-  return yargs.command(
-    'create <name> <template>',
-    'Creates a new JSS application based on one of the sample apps.',
-    (innerBuilder) =>
-      innerBuilder
-        .positional('name', {
-          type: 'string',
-          describe: 'The name of the app to create. ',
-        })
-        .positional('template', {
-          type: 'string',
-          describe:
-            'The template to create the app from; corresponds to folders in https://github.com/Sitecore/jss/tree/master/samples',
-        })
-        .option('hostName', {
-          requiresArg: false,
-          type: 'string',
-          describe:
-            'Sets the host name of the Sitecore site if this app is deployed to Sitecore. Defaults to \'$name.dev.local\'',
-        })
-        .option('repository', {
-          requiresArg: false,
-          type: 'string',
-          alias: 'r',
-          default: 'Sitecore/jss',
-          describe:
-            'Configures the GitHub repository to get the app from. Used to create from your own custom templates. Ex: myOrg/myRepository',
-        })
-        .option('branch', {
-          requiresArg: false,
-          type: 'string',
-          alias: 'b',
-          default: 'master',
-          describe:
-            'Configures the GitHub branch to get the app from. Used to create from your own custom templates. Ex: beta',
-        })
-        .option('source', {
-          requiresArg: false,
-          type: 'string',
-          alias: 's',
-          describe:
-            'Sources the app template from a local filesystem path, instead of a GitHub repository. Good for private templates.',
-        })
-        .option('proxy', {
-          requiresArg: false,
-          type: 'string',
-          describe: 'Specify a HTTP proxy when downloading templates.',
-        }),
-    handler
-  );
-}
-
-/**
- * @param {any} argv
- */
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-async function handler(argv: any) {
-  const newProjectPath = path.join(process.cwd(), argv.name);
-  const createScriptPath = path.join(newProjectPath, 'jss-create.js');
-
-  argv.hostName = argv.hostName ? argv.hostName : `${argv.name}.dev.local`;
-
-  checkName(argv.name, newProjectPath);
+const getDeprecationMessage = (argv: { [key: string]: unknown } = {}) =>
+  [
+    chalk.red('**DEPRECATED**'),
+    'jss create is not supported from JSS CLI 20.0.0',
+    // TODO: add (link to docs: https://doc.sitecore.net/<officialLink> ) when article will be created
+    'Please use npm init',
+    `Use: npm init sitecore-jss ${argv?.template || 'nextjs'}`,
+    'Please downgrade to JSS CLI 19.0.0 if you want to install older releases',
+    'Use:',
+    'npm install -g @sitecore-jss/sitecore-jss-cli@19.0.0',
+    `jss create ${argv?.name || 'nextjs-app'} ${argv?.template ||
+      'nextjs'} --branch release/19.0.0`,
```

Comment:
I thought chalk highlighting wouldn't work, but I see you're using it for the deprecated message above. If we can use it, would be nice to highlight the commands in green (consistent with our next steps). Some language suggestions in here as well.

```suggestion
    'jss create is not supported from JSS CLI version 20.0.0.',
    // TODO: add (link to docs: https://doc.sitecore.net/<officialLink> ) when article will be created
    'Please use npm init sitecore-jss instead. For example:',
    chalk.green(`Use: npm init sitecore-jss ${argv?.template || 'nextjs'}`),
    'Please downgrade to a previous JSS CLI version if you want to install older releases. For example:',
    chalk.green('npm install -g @sitecore-jss/sitecore-jss-cli@19.0.0'),
    chalk.green(`jss create ${argv?.name || 'nextjs-app'} ${argv?.template ||
      'nextjs'} --branch release/19.0.0`),
```

Reply:
Good propose, done.